scripts/qa_crowbarsetup.sh

```diff
@@ -1742,10 +1752,17 @@ function onadmin_allocate()
             local nodes=(
                 $(get_all_discovered_nodes | head -n 3)
             )
-            for n in $(seq 1 2); do
-                echo "Setting node $(($n+1)) to SLE12 SP1 Storage..."
-                set_node_role_and_platform ${nodes[$n]} "storage" "suse-12.1"
-            done
+            if iscloudver 6 && [[ $cloudsource =~ ^M[1-7]$ ]]; then
+                for n in $(seq 1 2); do
+                    echo "Setting node $(($n+1)) to SLE12 Storage..."
+                    set_node_role_and_platform ${nodes[$n]} "storage" "suse-12.0"
+                done
+            else
+                for n in $(seq 1 2); do
+                    echo "Setting node $(($n+1)) to SLE12 SP1 Storage..."
+                    set_node_role_and_platform ${nodes[$n]} "storage" "suse-12.1"
```

Comment:
IMHO too much code duplication. why not just use a storage_os=suse-12.1 variable (the exact echo message does not matter)